### PR TITLE
Bug fix for #2064: eliminated false error message

### PR DESCRIPTION
**Pull request by Snickermicker**
Eliminated false error message. Improved log output.

### CODE REVIEW

File: bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/bus/KNXBusReaderScheduler.java [hunk 1]
```diff
@@ -182,7 +185,7 @@ public synchronized boolean scheduleRead(Datapoint datapoint, int autoRefreshTim
 		if (oldListNumber > 0) { 
 			if (oldListNumber==autoRefreshTimeInSecs) {
 				sLogger.debug("Datapoint '{}' was already in  auto refresh list {}", datapoint.getName(), autoRefreshTimeInSecs);
-				return false;
+				return true;
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/bus/KNXBusReaderScheduler.java]: why did you change the logic here? There doesn't seem to be complains about an erroneous behaviour?

**Comment by Snickermicker** [comment]
Yes, there was a complaint: #2064. The logic caused a subsequent error message when a record was already in a list and would've been overwritten.
This function shouldn't fail, since the whole purpose for the function is to add a record to the list and if it already existed then it's still OK.
